### PR TITLE
Feature/CORE-172/use single menu for branching category

### PR DESCRIPTION
**Pull request by safydy-r**
Overview:
core-172

### CODE REVIEW

File: src/components/interaction-designer/toolbar/TreeBuilderToolbar.vue [hunk 1]
```diff
@@ -235,39 +235,50 @@
...
+                <a v-for="(classDetails, className) in blockClassesForBranchingCategory"
+                   :key="className + 'item'"
+                   class="nav-link single-menu"
+                   href="#"
+                   @click.prevent="handleAddBlockByTypeSelected(classDetails)">
+                  {{ 'flow-builder.branching' | trans }}
+                </a>
+              </li>
+              <li
+                v-else
```
Comment (safydy-r) [inline src/components/interaction-designer/toolbar/TreeBuilderToolbar.vue:249]: No change for this section apart from the `v-else`

File: src/components/interaction-designer/toolbar/TreeBuilderToolbar.vue [hunk 2]
```diff
@@ -235,39 +235,50 @@
...
-                    :data-default-num-connections="classDetails['defaultConnections']"
-                    @click.prevent="handleAddBlockByTypeSelected(classDetails)">
-                    {{ translateTreeClassName(className) }}
-                  </a>
-                </template>
-                <slot name="extra-dropdown-items-for-category3" />
-              </div>
-            </li>
+            <template v-if="!isEmpty(blockClassesForBranchingCategory)">
+              <li class="nav-item" v-if="Object.keys(blockClassesForBranchingCategory).length === 1">
```
Comment (safydy-r) [inline src/components/interaction-designer/toolbar/TreeBuilderToolbar.vue:239]: added this to display single menu if the branching category only has 01 element

File: src/components/interaction-designer/toolbar/TreeBuilderToolbar.vue [hunk 3]
```diff
@@ -235,39 +235,50 @@
...
+                <a v-for="(classDetails, className) in blockClassesForBranchingCategory"
```
Comment (safydy-r) [inline src/components/interaction-designer/toolbar/TreeBuilderToolbar.vue:240]: even having only 01 element, I looped with v-for to get the content easily
Note: blockClassesForBranchingCategory is an associative array

File: src/components/interaction-designer/toolbar/TreeBuilderToolbar.vue [hunk 4]
```diff
@@ -720,13 +731,13 @@ export default TreeBuilderToolbar
 .tree-workspace-panel-heading-contents .nav .nav-item.show>a:hover,
 .tree-workspace-panel-heading-contents nav .nav-item.show>a:focus {
   color: #ffffff;
-  background-color: #8C215C;
+  background-color: $primary-600;
```
Comment (safydy-r) [inline src/components/interaction-designer/toolbar/TreeBuilderToolbar.vue:734]: took the opportunity to use variables instead

**Review by object-required — APPROVED** [review]
Great improvement to the UI 💯

**Review by bulatgab — APPROVED** [review]

**Pull request merged**